Sources/SwiftParser/Parser.swift

```diff
@@ -585,3 +585,24 @@ extension SyntaxText {
     try body(UnsafeBufferPointer<UInt8>(start: self.baseAddress, count: self.count))
   }
 }
+
+extension Parser {
+  /// Consume a period as part of a member expression/type. Create unexpected +
+  /// missing tokens if there's trailing whitespace no leading whitespace
```

Comment:
there's trailing whitespace **and** no leading whitespace.

---

Do we also need to diagnose the case that is leading whitespace but no trailing whitespace?

Reply:
Leading and no trailing is fine, happens all the time with newlines:
```
foo
  .bar
  ```